lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -91,6 +91,7 @@ def __init__(self, *args, **kwargs):
         self._last_poll_time = datetime.now()
         self._last_report_timestamp = datetime.now()
         self._bridge_name = kwargs['config']['bridge_name']
+        self._periodic_stats_reporting = kwargs['config'].get('periodic_stats_reporting', True)
```

Comment:
This will change to 
        self._periodic_stats_reporting = kwargs['config']['enforcement'].get('periodic_stats_reporting', True)